en/micro-integrator/docs/use-cases/tutorials/exposing-several-services-as-a-single-service.md

```diff
@@ -573,8 +573,8 @@ You can now start updating the API resource with the mediation flow.
                    <li>Select <strong>Expression</strong> from the list.
                     </li>
                    <li>
-                      Enter "$ctx:uri.var.appointment_id".
-                      Note that the "$ctx" method is similar to using the <code>get-property</code> method. This method checks in the message context.
+                      Enter <code>$ctx:uri.var.appointment_id<code>.
```

Comment:
I don't think the $ sign renders properly in the doc build when we use the <code> tags.
That is why we have used double quotes.

Reply:
Though we have double quotes, it is not rendering properly in the current doc. Do we have any other alternative?

Reply:
Have you tried with single backticks? That should work.

Reply:
Fixed in bfae56b70733d657700734f5eccdb02547feea44